pkg/cloud/vsphere/services/govmomi/net/net.go

```diff
@@ -97,3 +92,25 @@ func GetNetworkStatus(
 
 	return allNetStatus, nil
 }
+
+// IsExternalIPAddr returns a nil error if the provided IP address is valid and
+// can be used externally to the VM.
+func IsExternalIPAddr(addr string) error {
```

Comment:
Nit:
If possible can we name this method differently?
like `IsNonLocalIPAddr(addr string)`

`External IP` seems to have a different meaning in my mind and may cause confusion to some.

Reply:
Hi @sidharthsurana,

Please see above. The meaning is documented in the function's GoDoc.

Reply:
Hi @andrewsykim and @sidharthsurana,

FWIW, I originally called this `IsNonLinkLocal`, but I test for other things too. Hence the flip on the name.

Reply:
Hi @andrewsykim and @sidharthsurana,

Please check now and see what you think.